core/web/user_controller.go

```diff
@@ -101,6 +107,7 @@ func (c *UserController) DeleteAPIToken(ctx *gin.Context) {
 		return
 	}
 	{
+		c.App.GetLogger().Audit(logger.API_TOKEN_DELETED, map[string]interface{}{"user": user.Email})
```

Comment:
nit/ I see a `"email": user.Email` down below - should we align these one way or the other?

Reply:
Updated all instances to "user" 🙂